README.md

```diff
@@ -110,31 +117,11 @@ zlvals = [0.02 0.035 0.05 0.065 0.08 0.095 0.11 0.125 0.14 0.155 0.17 0.185 0.2
 
 ```
 
-### Alternative Units in Config Files
-
-By default the code works in units of \Delta\Sigma (Msolar/pc^2) vs radius in
-Mpc.  You can set the units of radius with "r_units" and the units for shear
-with "shear_units"
-
-To measure tangential shear vs radius in arcminutes
-```
-r_units     = "arcmin"
-shear_units = "shear"
-```
-You can even measure \Delta\Sigma vs radius in arcminutes
-```
-r_units     = "arcmin"
-shear_units = "deltasig"
-```
-
-In sigmacrit_style="sample", the output is always in shear units. The weighted inverse
-value of sigma_crit in each radial bin is written in an additional column and can be used 
-to convert the mean shears to \Delta\Sigma.
 
 Format of Lens Catalogs
 -----------------------
 
-The format is white-space delimited ascii.  The columns are
+The format is white-space delimited ascii with no newline at the end.  The columns are
```

Comment:
ahh, I misunderstood.  What you are saying is there should not be a final new line at the very end of the file.  Yes, this is true.